hail/src/main/scala/is/hail/expr/ir/PruneDeadFields.scala

```diff
@@ -1159,6 +1279,12 @@ object PruneDeadFields {
           upcastTable(rebuild(c, memo), requestedType, upcastGlobals = false)
         }
         TableUnion(rebuilt)
+
+      // REMOVE DEBUG
```

Comment:
?

Reply:
oops, yes should be removed. Needed this to debug some other rule that went wrong.